src/offline/cable_checks.F90

```diff
@@ -204,6 +205,8 @@ MODULE cable_checks_module
 
   TYPE(ranges_type), SAVE :: ranges
 
+  IF (cable_user%GW_MODEL) ranges%sucs = [30., 800.]   ! MMY the range [-0.8, -0.03] doesn't suit Mark Decker's version
```

Comment:
Can't be done in the module definition part. This will require some more in-depth changes that will be done later.